cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -151,22 +151,154 @@ describe('Moderation Tools for Posts', () => {
         });
       });
     });
+
+    context('when suspending user', () => {
+      beforeEach(() => {
+        cy.get('@moderatorUser').then((user) => {
+          cy.loginAndVisit(user, '/series_user/series-test-article-slug');
+          cy.findByRole('heading', { level: 1, name: 'Series test article' });
+          cy.findByRole('button', { name: 'Moderation' }).click();
+        });
+      });
+
+      it('should show Suspend User button', () => {
+        cy.getIframeBody('[title="Moderation panel actions"]').within(() => {
+          cy.findByRole('button', { name: 'Open admin actions' })
+            .as('moderatingActionsButton')
+            .pipe(click)
+            .should('have.attr', 'aria-expanded', 'true');
+
+          cy.findByRole('button', {
+            name: 'Suspend series_user',
+          }).should('exist');
+        });
+      });
+
+      it('should not suspend the user when no reason given', () => {
+        cy.getIframeBody('[title="Moderation panel actions"]').within(() => {
+          cy.findByRole('button', { name: 'Open admin actions' })
+            .as('moderatingActionsButton')
+            .pipe(click)
+            .should('have.attr', 'aria-expanded', 'true');
+
+          cy.findByRole('button', {
+            name: 'Suspend series_user',
+          }).click();
+        });
+        cy.findByRole('dialog').within(() => {
+          cy.findByRole('button', { name: 'Submit & Suspend' }).click();
+
+          cy.get('#suspension-reason-error')
+            .contains('You must give a reason for this action.')
+            .should('exist');
+        });
+      });
+
+      it('should suspend the user when suspension reason given', () => {
+        cy.getIframeBody('[title="Moderation panel actions"]').within(() => {
+          cy.findByRole('button', { name: 'Open admin actions' })
+            .as('moderatingActionsButton')
+            .pipe(click)
+            .should('have.attr', 'aria-expanded', 'true');
+
+          cy.findByRole('button', {
+            name: 'Suspend series_user',
+          }).click();
+        });
+        cy.findByRole('dialog').within(() => {
+          cy.findByRole('textbox', { name: 'Note:' }).type(
+            'My suspension reason',
+          );
+
+          cy.findByRole('button', { name: 'Submit & Suspend' }).click();
+        });
+
+        cy.findByTestId('snackbar')
+          .contains('Success! series_user has been updated.')
+          .should('exist');
+      });
+    });
+
+    context('when unsuspending user', () => {
+      beforeEach(() => {
+        cy.get('@moderatorUser').then((user) => {
+          cy.loginAndVisit(user, '/suspended_user/suspended-user-article-slug');
+          cy.findByRole('heading', {
+            level: 1,
+            name: 'Suspended user article',
+          });
+          cy.findByRole('button', { name: 'Moderation' }).click();
+        });
+      });
+
+      it('should show Unsuspend User button', () => {
```

Comment:
What are your thoughts on not having a dedicated test for if the button exists?

I ask, because we're interacting with the button later on (e.g. in the `it('should not unsuspend the user when no reason given'` and `it('should unsuspend the user when reason given'`).  By not having the explicit test we'd reduce the computational cost of the test suite (e.g. don't spin up another relatively expensive integration test).

Reply:
@jeremyf I thought about this. In my mind, because it's the same button (and we're just toggling the text), I felt testing the existence of the `Suspend <user>` button was sufficient.

But I am not opposed to testing the `Unsuspend <user>` button's existence. 👍🏾

Reply:
Yeah I don't think we need the separate test, as if the button doesn't exist in subsequent tests we'll get a clear error message that says as much 👍